ansible-allinone-demo-vm/roles/opencast-allinone/templates/opencast-ingest.sh

```diff
@@ -43,7 +58,12 @@ curl -f -i -s -D - -o /dev/null -u ${LOGIN} \
   -F title="OCPR Demo" \
   -F description="Opencast quick Jira ticket and pull request creator, https://github.com/lkiesow/ocpr" \
   -F creator="Lars Kiesow" \
-  -F identifier=ID-ocpr-demo
+  -F identifier=ID-ocpr-demo \
+  -F license="CC-BY" \
+  -F acl='{"acl": {"ace": [
+    {"allow": true,"role": "ROLE_ADMIN","action": "read"},
+    {"allow": true,"role": "ROLE_ADMIN","action": "write"},
+    {"allow": true,"role": "ROLE_ANONYMOUS","action": "read"}]}}'
```

Comment:
Users with `ROLE_ADMIN` have access anyway (same complaint applies to other ingests):

```suggestion
  -F acl='{"acl": {"ace": [{"allow": true,"role": "ROLE_ANONYMOUS","action": "read"}]}}'
```